README.md

```diff
@@ -494,7 +494,7 @@ fastify.register(require('fastify-swagger'), {
     if (params) transformed.params = convert(params)
     if (body) transformed.body = convert(body)
     if (querystring) transformed.querystring = convert(querystring)
-    if (response) transformed.response = convert(response)
+    // if (response) transformed.response = convert(response)
```

Comment:
why did you have to comment this out?

Reply:
i do not think response schema is not support with joi, seen here (github.com//pull/144#issuecomment-456106235)

Reply:
And with it in transform, fastify fails.

Reply:
If it is not supported, it is better to remove it from the docs.

Reply:
have removed it then